content/akeneo-web-api.yaml

```diff
@@ -602,6 +602,184 @@ paths:
+                  },
+                  "code": "tshirt",
+                  "family_variant": "familyVariantA1",
+                  "parent": null,
+                  "categories": ["clothing", "tshirt"],
+                  "values": {
+                    "a_price": [
+                      {
+                        "locale": null,
+                        "scope": null,
+                        "data": [
+                          {
+                            "amount": "50.00",
+                            "currency": "EUR"
+                          }
+                        ]
+                      }
+                    ],
+                    "a_number_float": [
+                      {
+                        "locale": null,
+                        "scope": null,
+                        "data": "12.5000"
+                      }
+                    ],
+                    "a_localized_and_scopable_text_area": [
+                      {
+                        "locale": "en_US",
+                        "scope": "ecommerce",
+                        "data": "I like tshirt!"
+                      }
+                    ]
+                  },
+                  "created": "2017-10-04T18:04:10+02:00",
+                  "updated": "2017-10-04T18:04:10+02:00"
+                },
+                {
+                  "_links": {
+                    "self": {
+                      "href": "http:\/\/localhost\/api\/rest\/v1\/product-models\/trousers"
+                    }
+                  },
+                  "code": "trousers",
+                  "family_variant": "familyVariantA1",
+                  "parent": null,
+                  "categories": ["clothing", "trousers"],
+                  "values": {
+                    "a_price": [
+                      {
+                        "locale": null,
+                        "scope": null,
+                        "data": [
+                          {
+                            "amount": "50.00",
+                            "currency": "EUR"
+                          }
+                        ]
+                      }
+                    ],
+                    "a_number_float": [
+                      {
+                        "locale": null,
+                        "scope": null,
+                        "data": "12.5000"
+                      }
+                    ],
+                    "a_localized_and_scopable_text_area": [
+                      {
+                        "locale": "en_US",
+                        "scope": "ecommerce",
+                        "data": "I like trousers!"
+                      }
+                    ]
+                  },
+                  "created": "2017-10-04T18:04:10+02:00",
+                  "updated": "2017-10-04T18:04:10+02:00"
+                }
+              ]
+            }
+          }
+        400:
+          $ref: "#/responses/400Error"
+        401:
+          $ref: "#/responses/401Error"
+        406:
```

Comment:
It's missing the 422 error, here.

Reply:
There is no search available so 422 should never happen

Reply:
See API-32. You can have 422 if there are errors on the query parameters such as `page`.